tests/test_controller.py

```diff
@@ -542,6 +541,13 @@ def test_exception_during_short_client_token_initialization_is_stored(self):
         )
         registry_integration.username = "something"
         registry_integration.set_setting(AuthdataUtility.VENDOR_ID_KEY, "vendorid")
+        # Indicate that the library has successfully registered.
+        ConfigurationSetting.for_library_and_externalintegration(
+            self._db,
+            RegistrationConstants.LIBRARY_REGISTRATION_STATUS,
+            self.library,
+            registry_integration,
+        ).value = RegistrationConstants.SUCCESS_STATUS
```

Comment:
Existing tests were expecting an active integration , so here we set the needed status.